sklearn/svm/_classes.py

```diff
@@ -98,20 +102,24 @@ class LinearSVC(LinearClassifierMixin, SparseCoefMixin, BaseEstimator):
         will be ignored.
 
     fit_intercept : bool, default=True
-        Whether to calculate the intercept for this model. If set
-        to false, no intercept will be used in calculations
-        (i.e. data is expected to be already centered).
+        Whether or not to fit an intercept. If set to True, the feature vector
+        is extended to include an intercept term: ``[x_1, ..., x_n, 1]``, where
+        1 corresponds to the intercept. If set to False, no intercept will be
+        used in calculations (i.e. data is expected to be already centered).
 
     intercept_scaling : float, default=1.0
-        When self.fit_intercept is True, instance vector x becomes
-        ``[x, self.intercept_scaling]``,
-        i.e. a "synthetic" feature with constant value equals to
-        intercept_scaling is appended to the instance vector.
-        The intercept becomes intercept_scaling * synthetic feature weight
-        Note! the synthetic feature weight is subject to l1/l2 regularization
-        as all other features.
-        To lessen the effect of regularization on synthetic feature weight
-        (and therefore on the intercept) intercept_scaling has to be increased.
```

Comment:
I think this is a very important and intuitive statement which we should include here and other places, rather than to remove it.

Reply:
Absolutely, I have therefore reformulated it as: 

`To reduce the impact of the regularization on the intercept, the `intercept_scaling` parameter can be set to a value greater than 1.`

I think it's a little more precise, though less intuitive, I recon. `Increased` depends from where we start. Would we increase from 0 to 0.9, the regularization on the intercept would still be stronger instead of weaker. Only `intercept_scaling` larger than 1 would lessen the regularization.

Reply:
Then maybe:

```
To reduce the impact of the regularization on the intercept, the ``intercept_scaling`` parameter can be set to a value greater than 1; the higher the value of ``intercept_scaling``, the lower the impact of regularization on it.